doc/DEPENDENCIES.md

```diff
@@ -7,25 +7,28 @@ The DMOD project has two related sets of dependencies:
+| Docker Engine           | \>=20.10.16         |                      Docker Swarm support required (i.e., alternatives without this, like Podman, are insufficient)                    |
+| Docker Compose          | \>=2.0.x            |               [See issue #133](https://github.com/NOAA-OWP/DMOD/issues/133); _deployx_ plugin now required as noted below              |
 | _deployx_ Docker plugin | \>=0.0.1            | Available [here](https://github.com/aaraney/deployx).  Results in some transitive dependencies not explicitly enumerated here (e.g., Go) |
-| Bash                    | \>=3.2.57           |                                                                                                                                          |
-| OpenSSL / LibreSSL      | \>=3.0.0 / \>=2.8.3 |                                                                                                                                          |
+| Bash                    | \>=3.2.57           |                                                                                                                                        |
+| OpenSSL / LibreSSL      | \>=3.0.0 / \>=2.8.3 |                                                                                                                                        |
+| MinIO CLI client        |  |                                                                                                                                        |
 
 ## Development Dependencies
-| Dependency                                | Constraints                                 |                       Notes                        |
-|-------------------------------------------|:--------------------------------------------|:--------------------------------------------------:|
-| [Usage Dependencies](#usage-dependencies) | Same as above                               |                                                    |
-| Python                                    | \>=3.8.x                                    |                                                    |
-| Python Packages                           | See [requirements.txt](../requirements.txt) | Recommend installing in Python virtual environment |
+| Dependency                                | Constraints                                 |                                 Notes                                  |
+|-------------------------------------------|:--------------------------------------------|:----------------------------------------------------------------------:|
+| [Usage Dependencies](#usage-dependencies) | Same as above                               |                                                                        |
+| Python                                    | \>=3.8.x                                    |                                                                        |
```

Comment:
Might want to bump that up to 3.9.x - we've got 3.9 _somewhere_ in the pipeline (don't remember where). It's probably the github action.

Reply:
The (perhaps unofficial) rule of thumb has been for DMOD to support all the versions of Python that are currently, officially supported, which includes 3.8 for now.  Given ecosystems like RHEL's that aren't as quick to move to new versions, I think this could make a difference in community adoption (which this particular PR is also geared toward).

I could have sworn we had a matrix in the Github Actions to support testing on a handful of different Python versions, including 3.8 and 3.9.  Maybe I'm mixing that up with somewhere else and we should add it here.

Regardless, if there is something specific for which we need to consider breaking with the current approach, then let's open a separate issue and formally consider it.

Reply:
> I could have sworn we had a matrix in the Github Actions to support testing on a handful of different Python versions, including 3.8 and 3.9. Maybe I'm mixing that up with somewhere else and we should add it here.

Ah, looks like I was remembering what Nels had set up in the ngen-cal project.  We probably should adopt something similar.